cli/connect/console.go

```diff
@@ -248,6 +255,11 @@ func getCompleter(console *Console) prompt.Completer {
 			return nil
 		}
 
+		getSuggestionsFuncBody, err := static.GetStaticFileContent(ConnectLuaTemplateFS, "suggestions_func_body.lua")
```

Comment:
The same as above - can be computed in `init()`.

See, you compute the same thing on each `comleter` call, it is called for each new symbol received by user.